js/post-select/containers/term-select-form-field.js

```diff
@@ -50,15 +51,20 @@ class TermSelect extends React.Component {
 			path: addQueryArgs( `wp/v2/${restBase}/`, query ),
 			signal: this.fetchPostAbortController.signal,
 		} ).then( ( [ terms, headers ] ) => {
+			const { value } = this.props;
+
 			const newOptions = _uniqBy( [ ...options, ...terms.map( term => ( {
 				value: term.id,
 				label: term.name,
 			} ) ) ], 'value' );
 
+			const newValue = value.map( id => newOptions.find( option => option.value === id ) ).filter( Boolean );
```

Comment:
what does `filter( Boolean )` do

Reply:
It removes any falsy values such as `undefined`, which is the result of `find` if it didn't find anything.

Reply:
that's useful